src/index.html

```diff
@@ -1,14 +1,38 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
+
   <body>
     <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo">
+        <a href="#home"><img src="./images/logo.png" alt="logo"></a>
+      </div>
+      <nav class="header__nav nav">
+        <ul class="nav__list">
+          <li class="nav__item"><a href="#" class="nav__link is-active">APPLE</a></li>
```

Comment:
1. apply code styling recommendations from a checklist
2. use lowercased text at `html`. You can change it at css
```suggestion
          <li class="nav__item">
            <a href="#" class="nav__link is-active">
              apple
            </a>
          </li>
```